Files/UserControls/LayoutModes/PhotoAlbum.xaml

```diff
@@ -271,14 +271,14 @@
     <Grid
         Padding="0,0,0,0"
         VerticalAlignment="Stretch"
-        Background="{StaticResource ApplicationPageBackgroundThemeBrush}"
         ContextFlyout="{StaticResource BaseLayoutContextFlyout}">
         <ProgressBar
             Name="ProgBar"
             Height="10"
             VerticalAlignment="Top"
             IsIndeterminate="True"
-            Visibility="{x:Bind AssociatedViewModel.LoadIndicator.IsVisible, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+            Visibility="{x:Bind AssociatedViewModel.LoadIndicator.IsVisible, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+            Background="Transparent" />
```

Comment:
Does setting a transparent background here fix anything? Just curious...

Reply:
This is not my changes. You can ignore that line.